<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>db_settings_advanced</class>
 <widget class="QDialog" name="db_settings_advanced">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advanced database settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_db_user">
     <property name="text">
      <string>Database user:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_db_user_pass">
     <property name="text">
      <string>User password:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="edit_db_user_pass"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="edit_db_user"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_db_polling">
     <property name="minimumSize">
      <size>
       <width>233</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Database polling interval(ms):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="edit_db_polling"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_auth_db_name">
     <property name="text">
      <string>Authentication db name:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="edit_auth_db_name"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edit_db_user</tabstop>
  <tabstop>edit_db_user_pass</tabstop>
  <tabstop>edit_db_polling</tabstop>
  <tabstop>edit_auth_db_name</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>db_settings_advanced</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>db_settings_advanced</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
